code/game/machinery/doors/sentencing.dm

```diff
@@ -0,0 +1,586 @@
+		return .
+
+	if( !istype( corp_regs ))
+		. += "There was an error loading corporate regulations, please <a href='?src=\ref[src];button=change_menu;choice=main_menu'>Try Again</a>"
+		return .
+
+	. += charges_header()
+	. += "<hr>"
+	switch( menu_screen )
+		if( "low_severity" )
+			. += low_severity()
+		if( "med_severity" )
+			. += med_severity()
+		if( "high_severity" )
+			. += high_severity()
+
+	. += "<br><hr>"
+	. += "<center><a href='?src=\ref[src];button=change_menu;choice=incident_report'>Return</a></center>"
+
+/obj/machinery/computer/sentencing/proc/charges_header()
+	. = "<center>"
+
+	if( menu_screen == "low_severity" )
+		. += "Low Severity"
+	else
+		. += "<a href='?src=\ref[src];button=change_menu;choice=low_severity'>Low Severity</a>"
+
+	. += " - "
+
+	if( menu_screen == "med_severity" )
+		. += "Medium Severity"
+	else
+		. += "<a href='?src=\ref[src];button=change_menu;choice=med_severity'>Medium Severity</a>"
+
+	. += " - "
+
+	if( menu_screen == "high_severity" )
+		. += "High Severity"
+	else
+		. += "<a href='?src=\ref[src];button=change_menu;choice=high_severity'>High Severity</a>"
+
+	. += "</center>"
+
+	return .
+
+/obj/machinery/computer/sentencing/proc/low_severity()
+	. = ""
+
+	// Low severity
+	. += "<table class='border'>"
+	. += "<tr>"
+	. += "<th colspan='5'>Misdemeanors</th>"
+	. += "</tr>"
+
+	. += "<tr>"
+	. += "<th>Name</th>"
+	. += "<th>Description</th>"
+	. += "<th>Brig Sentence</th>"
+//	. += "<th>Fine</th>"
+	. += "<th>Button</th>"
+	. += "</tr>"
+
+	for( var/datum/law/L in corp_regs.low_severity )
+		. += "<tr>"
+		. += "<td><b>[L.name]</b></td>"
+		. += "<td><i>[L.desc]</i></td>"
+		. += "<td>[L.min_brig_time] - [L.max_brig_time] minutes</td>"
+//		. += "<td>$[L.min_fine] - $[L.max_fine]</td>"
+		. += "<td><a href='?src=\ref[src];button=add_charge;law=\ref[L]'>Charge</a></td>"
+		. += "</tr>"
+
+	. += "</table>"
+
+	return .
+
+/obj/machinery/computer/sentencing/proc/med_severity()
+	. = ""
+
+	// Med severity
+	. += "<table class='border'>"
+	. += "<tr>"
+	. += "<th colspan='5'>Indictable Offences</th>"
+	. += "</tr>"
+
+	. += "<tr>"
+	. += "<th>Name</th>"
+	. += "<th>Description</th>"
+	. += "<th>Brig Sentence</th>"
+	. += "<th>Prison Sentence</th>"
+//	. += "<th>Fine</th>"
+	. += "<th>Button</th>"
+	. += "</tr>"
+
+	for( var/datum/law/L in corp_regs.med_severity )
+		. += "<tr>"
+		. += "<td><b>[L.name]</b></td>"
+		. += "<td><i>[L.desc]</i></td>"
+		. += "<td>[L.min_brig_time] - [L.max_brig_time] minutes</td>"
+		. += "<td><a href='?src=\ref[src];button=add_charge;law=\ref[L]'>Charge</a></td>"
+		. += "</tr>"
+
+	. += "</table>"
+
+	return .
+
+/obj/machinery/computer/sentencing/proc/high_severity()
+	. = ""
+
+	// High severity
+	. += "<table class='border'>"
+	. += "<tr>"
+	. += "<th colspan='5'>Capital Offences</th>"
+	. += "</tr>"
+
+	. += "<tr>"
+	. += "<th>Name</th>"
+	. += "<th>Description</th>"
+	. += "<th>Brig Sentence</th>"
+	. += "<th>Prison Sentence</th>"
```

Comment:
That should be removed since you commented this out:
`"<td>[L.min_prison_time] - [L.max_prison_time] days</td>"`